src/main/java/com/SAS/LeagueManagement/LeagueManagementController.java

```diff
@@ -0,0 +1,226 @@
+        if (league == null || season == null) {
+            return false;
+        }
+
+        if (rankPolicyId.length() == 0) {
+            return false;
+        }
+
+        LeagueRankPolicy rankPolicy;
+
+        //TODO: check permissions
+        switch (rankPolicyId) {
+            case "1":
+                rankPolicy = new NumberOfWins(league,season);
+                league.addRankPolicy(season,rankPolicy);
+                season.addRankPolicy(league,rankPolicy);
+                break;
+
+            case "2":
+                rankPolicy = new GoalDifference(league,season);
+                league.addRankPolicy(season,rankPolicy);
+                season.addRankPolicy(league,rankPolicy);
+                break;
+
+            default:
+                System.out.println("No policy was found with this id");
+                return false;
+        }
+
+        System.out.println("The policy was added successfully");
+        return true;
+    }
+
+    /**
+     * The function receives league, season and points policy id and set the points policy to the league according to the season
+     * @param league
+     * @param season
+     * @param pointsPolicyId
+     * @return
+     */
+    public boolean addPointsPolicy(League league, Season season, String pointsPolicyId){
+        if (league == null || season == null) {
+            return false;
+        }
+
+        if (pointsPolicyId.length() == 0) {
+            return false;
+        }
+
+        PointsPolicy pointsPolicy;
+
+        //TODO: check permissions
+        switch (pointsPolicyId) {
+            case "1":
+                pointsPolicy = new OnePointForWinAndNoneForDraw(league,season);
+                league.addPointsPolicy(season,pointsPolicy);
+                season.addPointsPolicy(league,pointsPolicy);
+                break;
+
+            case "2":
+                pointsPolicy = new TwoForWinOneForDraw(league,season);
+                league.addPointsPolicy(season,pointsPolicy);
+                season.addPointsPolicy(league,pointsPolicy);
+                break;
+
+            case "3":
+                pointsPolicy = new ThreeForWinOneForDrawPolicy(league,season);
+                league.addPointsPolicy(season,pointsPolicy);
+                season.addPointsPolicy(league,pointsPolicy);
+                break;
+
+            default:
+                System.out.println("No policy was found with this id");
+                return false;
+        }
+
+        System.out.println("The policy was added successfully");
+        return true;
+    }
+
+    /**
+     * The function receives league, season and game policy id and set the game policy to the league according to the season
+     * @param league
+     * @param season
+     * @param gamePolicyId
+     * @return
+     */
+    public boolean addGamePolicy(League league, Season season, String gamePolicyId){
+        if (league == null || season == null) {
```

Comment:
Maybe add a function of validate in each function.

Reply:
concider doing it in feauture task, right now its ok